openmeter/customer/customer.go

```diff
@@ -68,53 +87,9 @@ func (i CreateCustomerInput) Validate(_ context.Context, _ Service) error {
 	return nil
 }
 
-// DeleteCustomerInput represents the input for the DeleteCustomer method
```

Comment:
So I agree that there is not need for the `namespace` info for `Get/Delete` operations since the the ID must be unique anyway. However I'd keep the `GetCustomerInput/DeleteCustomerInput` structure even if there is a single `ID` attribute for now, but it might change later on where we want to add additional configuration like how get/delete needs to be performed, etc.
Although I have no strong preference it might be a good idea to use a single pattern for service/repo interfaces.

Tbh, I am not sure how the ccurent change is any better: https://github.com/openmeterio/openmeter/pull/1542/files#diff-89424f051e363b1634f7c3a49f4d8c07449b194247eb9fb2b206cb39aee43305R107

Reply:
@turip @hekike ^

Reply:
I prefer that if a single argument is passed in (especially if it is an ID), then we don't wrap it. If we need more than one parameter, we should wrap it up and update the depending code.

This saves some boilerplate code, but I can change it if you think that's a better approach.